assets/css/wiki.css

```diff
@@ -82,3 +66,19 @@ details > summary {
   color: #1882A1;
   cursor: pointer;
 }
+
```

Comment:
If the styling for `cc` and `asio` divs is intentionally identical, can it not be declared once in a more general stylesheet, or is there only one per page?

Reply:
Think there has to be one per page (at least, there's wiki.css, home.css and kb.css)

Reply:
There’s also a general css file which gets included in every page. I think it’s fw.css.